.github/workflows/lint.yaml

```diff
@@ -53,19 +53,12 @@ jobs:
           fetch-depth: 0
       - name: Run Checkton
         id: checkton
-        uses: chmeliik/checkton@v0.1.2 
+        uses: chmeliik/checkton@v0.2.2
         # Migrating to the konflux-ci org
         with:
           fail-on-findings: true
-          find-copies-harder: true
-        continue-on-error: true 
-      - name: View SARIF Checkton Results 
-        run: cat "${{ steps.checkton.outputs.sarif }}"
-      - name: Check if Checkton failed
-        run: |
-          if [ "${{ steps.checkton.outcome }}" != "success" ]; then
-            echo "Checkton found issues"
-            exit 1
-          else
-            echo "Checkton found no issues"
-          fi
+          find-copies-harder: false
+          # Set to false to disable the differential check
+          differential: false 
+          # Add a regex to include only the files you want to check
+          include-regex: "tasks/check-data-keys/.*\\.yaml"
```

Comment:
Opening a thread to make sure this is not left here. (Also, missing a newline at the end.)